File: ProAgil.Domain/Palestrante.cs
using System.Collections.Generic;
namespace ProAgil.Domain
{
    public class Palestrante
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public int MiniCurriculo { get; set; }
        public string ImagemUrl { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public List<RedeSocial> RedesSociais { get; set; }
    }
}

File: ProAgil-App/src/app/_models/Palestrante.ts
import { RedeSocial } from './RedeSocial';
import { Evento } from './Evento';


export interface Palestrante {


    id: number;
    nome: string;
    miniCurriculo: number;
    imagemUrl: string;
    telefone: string;
    email: string;
    redeSociais: RedeSocial[];
    PalestranteEventos: Evento[];

}
